core/src/main/java/com/sopt/core/designsystem/component/button/NoostakButton.kt

```diff
@@ -0,0 +1,38 @@
+package com.sopt.core.designsystem.component.button
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.material3.Button
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.theme.NoostakTheme
+
+@Composable
+fun NoostakButton(
+    text: String,
+    textColor: Color,
+    buttonColor: Color,
+    onButtonClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    isEnabled: Boolean = true
```

Comment:
P2 : 보통 버튼들은 base가 비활 상태이기 때문에 기본값을 false로 설정하는 것이 더 효율적일 것 같아요
```suggestion
    isEnabled: Boolean = false
```